### PR TITLE
Feature/basic UI

### PR DESCRIPTION
**Pull request by NataliaPoletaeva**
In this PR, I have:
- created a file for the console performance;
- included all the necessary functions to be displayed on the ui.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    print 'Specialization '
+    specialization = gets.chomp
+
+    @people << Teacher.new(age, specialization, name)
+    puts 'Person created successfully'
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) a teacher(2)? [Input tNhe number]:'
```
Comment (hiromataba) [inline app.rb]: - [ ] [OPTIONAL] Please fix the typo, i think you wanted to write `the` not `tNhe`

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @NataliaPoletaeva  :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your app is working well :+1: 
- Very professional README files for both front and backend repo :+1: 
- Nice commit history :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please rename your `index.rb` to `main.rb` and in Your README file please add instructions on how to run your app, after renaming index.rb to main.rb this is the command you can use in your Instruction on how to run your app `ruby main.rb`

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NataliaPoletaeva** [comment]
Update: I don't know why but the linters here say that I don't have script permission even tho the shebang line is there. If you can, please give me a hint on how to fix it :)

**Comment by hiromataba** [comment]
Hello @NataliaPoletaeva Please use `rubocop -A` to auto fix your linters. i have done it locally on your project and it works 

image

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @NataliaPoletaeva,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**